ui_src/src/domain/stationOverview/hooks/reducer.js

```diff
@@ -29,6 +29,11 @@ const Reducer = (stationState, action) => {
                 ...stationState,
                 stationSocketData: updatedSocketState
             };
+        case 'SET_MESSAGES':
```

Comment:
Why don't you use "stationSocketData.messages"? It seems you are duplicating the messages.